easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-foss-2016b-Boost-1.63-Python-2.7.12.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+separate_build_dir = True
```

Comment:
the `easyblock` line is usually on top (if it's there), so keep that where it is now, just move the `separate_build_dir` down